models/articles-models.js

```diff
@@ -12,3 +12,19 @@ ORDER BY articles.created_at DESC;`;
     return rows;
   });
 };
+
+exports.fetchArticleComments = (article_id) => {
+  if (typeof article_id !== 'number' || isNaN(article_id))
+    return Promise.reject('Invalid article ID');
+  return db
+    .query('SELECT * FROM articles WHERE article_id = $1', [article_id])
```

Comment:
This absolutely works and is a great solution to check whether the articleID is valid.

If we were looking at writing production code here I'd probably say that running the comments query first and then the article's query after if the rows are empty may be a slightly more performant way of doing this as it means we only make 2 calls to the database if the comments are empty or the ID is invalid rarther than 2 calls to the database for every successful request.